test/lib/job.test.js

```diff
@@ -305,7 +304,7 @@ describe('Job Model', () => {
         });
 
         it('fail if getBuilds returns error', () => {
-            buildFactoryMock.list.rejects('error');
+            buildFactoryMock.list.rejects(new Error('error'));
```

Comment:
In `sinon-as-promised`, if you call `rejects` with a string it turns it into an error with the string as the message. `sinon` turns it into an error with the type being the string provided, so if we're checking against messages we need to construct the error object ourselves.